src/components/SlippageControl/CustomSlippageInput.tsx

```diff
@@ -148,7 +153,9 @@ const CustomSlippageInput: React.FC<Props> = ({ rawSlippage, setRawSlippage, isW
       return
     }
 
-    if (parsedValue > MAX_SLIPPAGE_IN_BIPS) {
+    const maxSlippage = isDegenMode ? MAX_DEGEN_SLIPPAGE_IN_BIPS : MAX_NORMAL_SLIPPAGE_IN_BIPS
+    if (parsedValue > maxSlippage) {
+      setTooltip(t`Max is ${formatSlippage(maxSlippage)}`)
```

Comment:
we dont need a state to manage tooltip message here. Just a variable that we can infer from `rawText` 
```
const maxSlippage = isDegenMode ? MAX_DEGEN_SLIPPAGE_IN_BIPS : MAX_NORMAL_SLIPPAGE_IN_BIPS
const tooltip = parseSlippageInput(rawText) > maxSlippage ? t`Max is ${formatSlippage(maxSlippage)}` : "